timemachine/cpp/CMakeLists.txt

```diff
@@ -65,7 +65,13 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR}/eigen)
 include_directories(${CMAKE_CURRENT_BINARY_DIR}/${CUB_SRC_DIR})
 
 set_property(TARGET ${LIBRARY_NAME} PROPERTY CUDA_STANDARD 14)
+
+if (NOT CUDA_ARCH)
+  set(CUDA_ARCH "75")
```

Comment:
This is to allow `pip install .` to succeed without needing to first set e.g. `CMAKE_ARGS="-DCUDA_ARCH=75"`